src/views/game-board.view.js

```diff
@@ -0,0 +1,74 @@
+class GameBoardView {
+  static renderShipsPlacement(boardArray, callback) {
+    const boardHtml = GameBoardView.board(boardArray);
+    const boardEl = document.getElementsByTagName('human-board')[0];
+    boardEl.innerHTML = boardHtml;
+    boardEl.querySelectorAll('.b-row > div').forEach((el) => {
+      el.addEventListener('click', (ev) => {
+        const x = parseInt(ev.target.dataset.id[0], 10);
+        const y = parseInt(ev.target.dataset.id[1], 10);
+        const length = document.getElementById('ship-length').value;
+        callback(x, y, parseInt(length, 10));
+      });
+    });
+  }
+
+  static renderBattle(boardArray, playerType, callback) {
+    const boardHtml = GameBoardView.board(boardArray, playerType);
+    if (playerType === 'AI') {
+      const boardEl = document.getElementsByTagName('ai-board')[0];
+      boardEl.innerHTML = boardHtml;
+      boardEl.querySelectorAll('.b-row > div').forEach((el) => {
+        el.addEventListener('click', (ev) => {
+          const ds = ev.target.dataset.id ? ev.target.dataset : ev.target.parentElement.dataset;
+          const x = parseInt(ds.id[0], 10);
+          const y = parseInt(ds.id[1], 10);
+          callback({ x, y });
+          // this.battle(boardArray, playerType, callback);
```

Comment:
**Kindly remove commented code inside file, you can always use git history to access previous code right**:slightly_smiling_face: